src/oidc/authorize.ts

```diff
@@ -125,6 +125,7 @@ export const generateAuthorizationUrl = async (
  * @param prompt - indicates to the OP which flow the user should see - eg login or registration
  *          See https://openid.net/specs/openid-connect-prompt-create-1_0.html#name-prompt-parameter
  * @param urlState - value to append to the opaque state identifier to uniquely identify the callback
+ * @param loginHint - send connecting user login hint to OP
```

Comment:
I don't really understand this documentation?

Reply:
from the spec :

```suggestion
 * @param loginHint - hint to the OP about the login identifier the user might use to log in
```